test/settings/RegionSettings.spec.js

```diff
@@ -0,0 +1,60 @@
+import { getWrapper } from '../shared';
+import NavigationBar from '../../src/components/NavigationBar';
+import SettingsPanel from '../../src/components/SettingsPanel';
+import RegionSettings from '../../src/components/RegionSettingsPanel';
+import LinkLine from '../../src/components/LinkLine';
+import Button from '../../src/components/Button';
+
+let wrapper = null;
+let panel = null;
+let regionSettings = null;
+let store = null;
+beforeEach(async () => {
+  jasmine.DEFAULT_TIMEOUT_INTERVAL = 32000;
+  wrapper = await getWrapper();
+  store = wrapper.props().phone.store;
+  const navigationBar = wrapper.find(NavigationBar).first();
+  await navigationBar.props().goTo('/settings');
+  panel = wrapper.find(SettingsPanel).first();
+  const regionLinkLine = panel.find(LinkLine).at(1);
+  await regionLinkLine.props().onClick();
+  regionSettings = wrapper.find(RegionSettings).first();
+});
+
+describe('region settings', async () => {
+  test('initial state', () => {
+    expect(regionSettings.find('div.label').first().props().children).toEqual('Region');
+  });
+
+  test('button state', async () => {
+    const saveButton = regionSettings.find(Button).first();
+    expect(saveButton.props().disabled).toEqual(true);
+    const input = regionSettings.find('input.input').first();
+    await input.props().onChange({ currentTarget: { value: '853' } });
+    expect(saveButton.props().disabled).toEqual(false);
+  });
+
+  test('save', async () => {
+    const saveButton = regionSettings.find(Button).first();
+    const input = regionSettings.find('input.input').first();
+    await input.props().onChange({ currentTarget: { value: '853' } });
+    await saveButton.props().onClick();
```

Comment:
Is it better to use `simulate('click')` to trigger `onClick` than just call `onChange` directly? It can ensure 'onClick' can be trigger by click event.